extensions/notebook/README.md

```diff
@@ -2,6 +2,12 @@
 
 Welcome to the Notebook extension for Azure Data Studio! This extension supports core notebook functionality including configuration settings, actions such as New / Open Notebook, and more.
 
+## Books in Azure Data Studio
+
+Jupyter Book allows opening a single "Book" of related notebooks and markdown files. This is a preview feature and will work if you open any Book folder in Azure Data Studio.
+
+Download sample book from https://github.com/jupyter/jupyter-book and open folder in Azure Data Studio to get started. You can learn more about Books on the [Juyter Book homepage](https://jupyter.org/jupyter-book/intro.html).
```

Comment:
>Download sample [](start = 0, length = 15)

Maybe "Download a sample book" - that seems to work better grammar-wise. I'd run this by the PMs regardless to get their signoffs

Reply:
Had PM Alan look at the README

Reply:
To clarify, was the change made for "[Download sample book](https://github.com/jupyter/jupyter-book) and open folder"? 

Also, on second look, Juyter Book homepage is spelled incorrectly :)

Reply:
Yep, the change was made and thanks for catching the spelling error!